CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # Changelog
 
+## Version 10.0.0 - 2023-10-02
+
+- Added `create_transformation`
+- Added `list_transformations`
+- Added `delete_transformation`
+- Added Python 3.11 support
+- Removed Python 3.6 and 3.7 support due to having reached end of life
```

Comment:
Consider changing the wording. There seems to be something missing here. How about `Removed Python 3.6 and 3.7 support due to them having reached end of life` or `Removed Python 3.6 and 3.7 support as they have reached end of life`?